instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/db/SqlStatementInfo.java

```diff
@@ -28,5 +29,14 @@ public SqlStatementInfo mapTable(Function<String, String> mapper) {
   public abstract String getOperation();
 
   @Nullable
-  public abstract String getTable();
+  public abstract String getIdentifier();
+
+  @Nullable
+  public String getTable() {
+    String operation = getOperation();
+    if (operation != null && !operation.equals("CALL")) {
+      return getIdentifier();
+    }
+    return null;
+  }
```

Comment:
I'm not sure this is worth a public method for now (we're trying to minimize public API surface), this logic could live in the one caller that cares if its a table or not

Reply:
Hey @trask, do you prefer something like this?

`    super.onStart(attributes, parentContext, request);

    SqlStatementInfo sanitizedStatement = sanitizer.sanitize(getter.rawStatement(request));
    String operation = sanitizedStatement.getOperation();
    internalSet(attributes, SemanticAttributes.DB_STATEMENT, sanitizedStatement.getFullStatement());
    internalSet(attributes, SemanticAttributes.DB_OPERATION, operation);
    internalSet(attributes, dbTableAttribute, !SQL_CALL.equals(operation) ? sanitizedStatement.getIdentifier() : null);`

Alternatively we could in theory have a private getTable method in SQLClientAttributedExtractor if you want to go that route.

Reply:
Hey @trask, do you prefer something like this?

`    internalSet(attributes, SemanticAttributes.DB_OPERATION, operation);
    internalSet(attributes, dbTableAttribute, !SQL_CALL.equals(operation) ? sanitizedStatement.getIdentifier() : null);`

Alternatively we could in theory have a private getTable method in SQLClientAttributedExtractor if you want to go that route.

Reply:
i'd probably go with

```
if (!SQL_CALL.equals(operation)) {
  internalSet(attributes, dbTableAttribute, sanitizedStatement.getIdentifier());
}
```